packages/leancode_lint/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 8.1.0
+
+- Enable custom lint `avoid_single_child_in_multi_child_widgets` for `Column`, `Row`, `Flex`, and `Wrap`
```

Comment:
Feel free to suggest other ones, I can easily add them

Reply:
Maybe also `Stack`? Usually could be replaced by `Align` or stuff

Reply:
We could also detect equivalent sliver widgets — `SliverMainAxisGroup` and `SliverCrossAxisGroup`